src/core/core.controller.js

```diff
@@ -713,25 +713,22 @@ module.exports = function(Chart) {
 			// Built in Tooltips
 			if (tooltipsOptions.enabled || tooltipsOptions.custom) {
 				tooltip._active = me.tooltipActive;
+				tooltip._eventPosition = helpers.getRelativePosition(e, me.chart);
 			}
 
 			// Hover animations
 			if (!me.animating) {
 				// If entering, leaving, or changing elements, animate the change via pivot
-				if (!helpers.arrayEquals(me.active, me.lastActive) ||
```

Comment:
Unfortunately, yes. What do you think about detecting if there are changes in update and returning a Boolean. If there are changes, we redraw.

Alternatively, we factor the position code out to here and pass it to the tooltip before update.